docs/manual/manual_stock_led.rst

```diff
@@ -0,0 +1,251 @@
+Still in the Control Center, find the RGB LED parameters, and set ``RGB LED Blue`` and ``RGB LED Green`` to 0, ``RGB LED Red`` to 150, and 
+set ``RGB LED Update Color``. Assuming that RGB strobing is disabled (which is the default configuration), your LED should now be statically red.
+
+.. image:: ../_static/manual_images/leds/red_only_on.png
+
+The RGB LED is configured with the same default pattern as the white LED, 3 blinks on and off followed by a longer period off, and the pattern repeats. 
+So, when we set ``RGB LED Strobing Active`` to ``Active``, we will see the red LED start to blink.
+
+Strobing activation can be changed on the fly, and is configured on boot-up according to the value stored in the module's persistent memory.
+
+<GET A VIDEO OF THIS (MAYBE ANOTHER SCREEN CAP)>
+
+.. _strobing_period:
+
+Strobing Period
+******************
+The strobing period defines the time it takes the module to complete one full cycle through the strobing pattern. This parameter only has an effect when strobing is 
+active. Strobe period is defined in seconds. Let's take the default strobing pattern used for both the RGB and white LEDs (blink, blink, blink, off). 
+By default, the strobing period is configured to 2.5 seconds.
+
+Using the Control Center, configure the white LED to an intensity of 1%, and enable strobing
+
+.. image:: ../_static/manual_images/leds/white_on_strobing.png
+
+Notice that the first on-off blink occurs every 2.5 seconds. Now, set ``White LED Strobing Period`` to 1 second. You'll now see that the pattern restarts every second. 
+Finally, set the period to 5 seconds, and the pattern will now restart every 5 seconds.
+
+**Strobing period operates identically for both the white and RGB LEDs.**
+
+<GET A VIDEO OF THIS WHEN WE CAN>
+
+Strobing Pattern
+********************
+The strobing pattern parameter defines a 32-bit bitmask that defines 32 segments of where an LED is on or off. The time it takes to move through all 32 segments is 
+defined by the :ref:`strobing period <strobing_period>`, and each individual segment takes :math:`\frac{\text{Strobe Period}}{32}` seconds. A 1 in the bitmask dictates the LED be on during that 
+segment, a 0 indicates the LED is off during the segment.
+
+Suppose you would like your pattern to alternate being on and off after every two segments
+
+.. image:: ../_static/manual_images/leds/strobing_pattern_example.png
+
+In this case, your pattern is defined by the bitmask (in hex) 0xCCCCCCCC, a value of 3435973836. Suppose the strobe period is configured to 16 seconds, making each 
+segment 0.5 seconds. Therefore, with this pattern, the light will turn on and off every second. In this example, we will use the white LED, **but the same applies 
+identically to the RGB LED**
+
+.. image:: ../_static/manual_images/leds/strobe_pattern_ex_config.png
+
+<GET A NICE VIDEO OF THIS WHEN WE CAN>
+
```

Comment:
This is a good example, and a good description. It may just be nice to have one more, slightly more weird example. Something not so...symmetrical, just to help drive things home. These nice symmetrical examples make it hard to see the start and end of the pattern, since it all looks the same at any point, so something a bit more weird with an obvious start, middle bits, and end, may help make it really clear whats going on.